looprpc/client.proto

```diff
@@ -127,6 +127,16 @@ service SwapClient {
     */
     rpc InstantOutQuote (InstantOutQuoteRequest)
         returns (InstantOutQuoteResponse);
+
+    /* loop: `static newaddress`
+    NewAddress requests a new static address for loop-ins from the server.
+    */
+    rpc NewAddress (NewAddressRequest) returns (NewAddressResponse);
```

Comment:
So these two rpcs are refactored to be in the `swapClient` rather than in the separate `StaticAddressClient` where they resided before. Their purpose is as their comments suggest.

Overall static address and deposit info will be introduced in https://github.com/lightninglabs/loop/pull/721.

Reply:
see my previous comment. I mentioned previously offline that this is part of a small refactoring.

Reply:
Yes you are right, previously in the static address server, the context was clear and the names made sense in that context. Now I'd call it `NewStaticAddress` and `ListUnspentDeposits`.